cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -129,6 +129,9 @@ features:
       updated-label:
         plural: seconds ago
         singular: second ago
+      primaryLabelOne: DAY
```

Comment:
have a more descriptive label for this, if it's indicating day, then the key should be primaryLabelDay